classes/class-pmpro-subscription.php

```diff
@@ -1113,43 +1113,47 @@ public function save() {
 	}
 
 	/**
-	 * Cancels this subscription in PMPro and at the payment gateway.
+	 * Cancels the subscription at the payment gateway.
 	 *
-	 * @since TBD
+	 * IPNs/Webhooks should then trigger the cancellation of the subscription in the
+	 * database.
+	 *
+	 * Legacy: Falls back on calling the gateway's cancel() method if the gateway does not
+	 * support cancelling PMPro_Subscription objects specifically.
 	 *
-	 * @return bool True if the subscription was canceled successfully in the payment gateway.
```

Comment:
Would be good to keep the @return docblock and tweak the description slightly.

Reply:
Done